static/styles/zulip.css

```diff
@@ -1396,6 +1396,10 @@ td.pointer {
     }
 }
 
+.on_hover_topic_read {
+    opacity: 0.7;
+}
```

Comment:
Should this be instead adding this to the `always_visible_topic_edit` block above, so hover behavior is consistent?

Reply:
@timabbott  If we remove and let the be opacity: 0.2, the element becomes too faint. We need to set value for the opacity separately for ".on_hover_topic_read" to make sure that the element is visible.
Also  "opacity: 0.2" is perfect for other elements because the other elements are just icons without any text in them. But in case of "unread counter" we need to keep visibility of the text in mind.
<img width="187" alt="Screenshot 2022-05-12 at 01 34 52" src="https://user-images.githubusercontent.com/85362194/167937088-cfb7dab1-b718-4180-86a9-e77d97d47351.png">